### PR TITLE
[PROJECT SUBMISSION] ipfs-cidsail

### PR DESCRIPTION
**Pull request by acul71**
Create a project profile here in the ipfs/community repository using the project submission template here. Name the file name after your project and fill in as much information as you can. Use your project name as the file name so the community can find your project easily!

### CODE REVIEW

File: templates/project-submission-template.md [hunk 1]
```diff
@@ -1,52 +1,74 @@
...
 * Slack:  
 * Twitter:  
-* Discord:  
+* Discord:  lucap#5517
 * Telegram:  
 * WeChat:  
 
 ## How to Contribute
 <!--How can the community contribute to your project?-->
+Go to https://github.com/acul71/ipfs-cidsail and follow instruction to install. Check the app and report any bugs/anomalies opening an issue
```
Comment (acul71) [inline templates/project-submission-template.md:74]: https://github.com/ipfs/community/discussions/601

Comment (acul71) [inline templates/project-submission-template.md:74]: https://github.com/ipfs/community/discussions/601

**Review by autonome — CHANGES_REQUESTED** [review]
Hi, you have modified the submission template. You must instead create a *new* file in the `projects` directory.

**Comment by acul71** [comment]
Create ipfs-cidsail.md #602

**Pull request closed without merging**